gui/widgets/tab.cpp

```diff
@@ -119,6 +128,7 @@ int TabWidget::addTab(const String &title, const String &dialogName) {
 	if (newWidth < _minTabWidth)
 		newWidth = _minTabWidth;
 	newTab._tabWidth = newWidth;
+	_widthTillLastTab += newWidth;
```

Comment:
This wouldn't be a problem, but maybe it would be best if I were to wrap this around `if(useRTL)` also?